modules/math/js/src/main/scala/gsp/math/geom/jts/JtsShape.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math.geom
+
+import gsp.math.Offset
+import gsp.math.geom.jts.syntax.offset._
+import org.locationtech.jts.geom.Geometry
+
+
+/**
+ * JS / JTS implementation of Shape.
+ */
+final case class JtsShape(g: Geometry) extends Shape {
+
+  override def contains(o: Offset): Boolean =
+    g.contains(o.point)
+
+  override def area: Area =
+    Area.fromMicroarcsecondsSquared.getOption(g.getArea.round).getOrElse(Area.MinArea)
```

Comment:
Do we need JavaScript and JVM-specific implementations of `Shape` now?  The translation to AWT needs to be separated of course but the `Shape` implementation itself doesn't differ?

Reply:
I agree, we should preserve the common bits and put the AWT/SVG parts onto the jvm/js specific parts